tests/test_installer.py

```diff
@@ -281,29 +280,12 @@ def test_get_game_size_from_unzip(self, mock_subprocess):
 --------          -------  ---                            -------
 159236636         104883200  34%                            189 files
 """
-        mock_subprocess().communicate.return_value = [stdout, b"stderr"]
+        mock_subprocess().communicate.return_value = [stdout, "stderr"]
         installer_path = "/home/i/.cache/minigalaxy/download/Beneath a Steel Sky/beneath_a_steel_sky_en_gog_2_20150.sh"
         exp = 159236636
         obs = installer.get_game_size_from_unzip(installer_path)
         self.assertEqual(exp, obs)
 
-    @mock.patch('shutil.which')
-    @mock.patch('os.listdir')
-    def test_get_exec_line(self, mock_list_dir, mock_which):
```

Comment:
Why was this test removed?

Reply:
Because the method it tests doesnt exist anymore. If replaced the self-coded shell masking with shlex.join(cmd) in the caller of get_exec_line

Reply:
Makes sense, will close this one then